__tests__/workers/postUpdated.ts

```diff
@@ -593,3 +611,142 @@ it('should not update already approved post', async () => {
   expect(submission.id).toEqual(uuid);
   expect(submission.status).toEqual(SubmissionStatus.Accepted);
 });
+
+describe('on post create', () => {
+  beforeEach(async () => {
+    await saveFixtures(con, Source, [
```

Comment:
The main before each already has all sources do you need this specific one?

Reply:
ah, missed that, thanks for pointing it out 👍